src/FSISPH/SolidFSISPHEvaluateDerivatives.cc

```diff
@@ -492,9 +497,14 @@ secondDerivativesLoop(const typename Dimension::Scalar time,
         }
 
         // save our max pressure from the AV for each node
-        maxViscousPressurei = max(maxViscousPressurei, rhoi*rhoj * QPiij.diagonalElements().maxAbsElement());
-        maxViscousPressurej = max(maxViscousPressurej, rhoi*rhoj * QPiji.diagonalElements().maxAbsElement());
-
+        {
+          const auto Qi = rhoi*rhoj * QPiij.diagonalElements().maxAbsElement();
+          const auto Qj = rhoi*rhoj * QPiji.diagonalElements().maxAbsElement();
+          maxViscousPressurei = max(maxViscousPressurei, Qi);
+          maxViscousPressurej = max(maxViscousPressurej, Qj);
+          effViscousPressurei = volj * Qi * Wi;
+          effViscousPressurej = voli * Qi * Wj;
```

Comment:
These probably should be "+=" so that they are summed over all neighbors.

Reply:
ah right thank you for catching that!